CONTRIBUTING.md

```diff
@@ -76,16 +91,216 @@ By making a contribution to this project, I certify that:
     sign-off) is maintained indefinitely and may be redistributed
     consistent with this project or the open source license(s)
     involved.
- ```
+```
+
 We require that every contribution to OpenSearch is signed with a Developer Certificate of Origin. Additionally, please use your real name. We do not accept anonymous contributors nor those utilizing pseudonyms.
 
 Each commit must include a DCO which looks like this
 
 ```
 Signed-off-by: Jane Smith <jane.smith@email.com>
 ```
+
 You may type this line on your own when writing your commit messages. However, if your user.name and user.email are set in your git configs, you can use `-s` or `--signoff` to add the `Signed-off-by` line to the end of the commit message.
 
+Fill in your personal details as well as an email that is public on github:
+
+```
+git config --global user.name "FIRST_NAME LAST_NAME"
+git config --global user.email "MY_NAME@example.com"
+```
+
+_Note_: If you want to set up this data only for a specific repository then do it without the `--global` parameter:
+
+```
+git config user.name "FIRST_NAME LAST_NAME"
+git config user.email "MY_NAME@example.com"
+```
+
+## First pull request
```

Comment:
This section should be a separate document. Because it's generic, I'd suggest proposing a new document in https://github.com/opensearch-project/.github